src/OpenLoco/src/Ui/Window.h

```diff
@@ -229,15 +228,15 @@ namespace OpenLoco::Ui
             uintptr_t info;
         };
         uint8_t pad_85E[0x870 - 0x85E];
-        uint16_t currentTab = 0;                  // 0x870
-        uint16_t frameNo = 0;                     // 0x872
-        uint16_t currentSecondaryTab = 0;         // 0x874
-        ViewportConfig viewportConfigurations[2]; // 0x876
-        WindowType type;                          // 0x882
+        uint16_t currentTab = 0;
+        uint16_t frameNo = 0;
+        uint16_t currentSecondaryTab = 0;
+        ViewportConfig viewportConfigurations[2];
+        WindowType type;
         uint8_t pad_883[1];
-        CompanyId owner = CompanyId::null; // 0x884
+        CompanyId owner = CompanyId::null;
         uint8_t var_885 = 0xFF;
-        AdvancedColour colours[enumValue(WindowColour::count)]; // 0x886
+        AdvancedColour colours[enumValue(WindowColour::count)];
         int16_t var_88A;
         int16_t var_88C;
         sfl::small_vector<Widget, 16> widgets;
```

Comment:
Can we move this `widgets` property up? It seems too important to be all the way at the bottom. Especially now that `_widgets` is gone, we're definitely letting go of the vanilla structure, anyway.

Reply:
We can and we probably should, but not just widgets but structs/classes are generally best when the largest things sit at the top.